eventuate-tram-quarkus-consumer-jdbc/src/main/java/io/eventuate/tram/quarkus/consumer/jdbc/TramConsumerJdbcConfiguration.java

```diff
@@ -15,6 +16,7 @@
 public class TramConsumerJdbcConfiguration {
 
   @Singleton
+  @UnlessBuildProperty(name = "transactional.noop.duplicate.message.detector.factory.enabled", stringValue = "true", enableIfMissing = true)
```

Comment:
Please explain the purpose of `transactional.noop.duplicate.message.detector.factory.enabled` and how it is used.

Reply:
Chris, here we have 2 DuplicateMessageDetectors: TransactionalNoopDuplicateMessageDetector and SqlTableBasedDuplicateMessageDetector. By default, when module is used as dependency SqlTableBasedDuplicateMessageDetector is created. To activate TransactionalNoopDuplicateMessageDetector is necessary to specify `transactional.noop.duplicate.message.detector.factory.enabled` property (however, I probably nee to change factory to configuration). The same way was used for micronaut: https://github.com/eventuate-tram/eventuate-tram-core/blob/wip-db-id-gen/eventuate-tram-micronaut-consumer-jdbc/src/main/java/io/eventuate/tram/micronaut/consumer/jdbc/TramConsumerJdbcFactory.java#L22 https://github.com/eventuate-tram/eventuate-tram-core/blob/master/eventuate-tram-micronaut-consumer-jdbc/src/main/java/io/eventuate/tram/micronaut/consumer/jdbc/TransactionalNoopDuplicateMessageDetectorFactory.java#L15